models/index.js

```diff
@@ -0,0 +1,36 @@
+import fs from 'fs';
+import path from 'path';
+import Sequelize from 'sequelize';
+
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+
+import dbConfig from `${__dirname}/../config/config.json`;
```

Comment:
There is an eslint error here. I gave feedback earlier today that you must ensure there are no eslint errors and your files are properly linted before asking for a review.

Reply:
> Also, ensure you maintain the project structure. All files and folders must be inside `src` folder

<img width="168" alt="Screen Shot 2019-12-04 at 17 15 47" src="https://user-images.githubusercontent.com/52411901/70154836-dcf49a00-16b9-11ea-8fdd-431f61524b27.png">
@adesege I think you are referring to those folders which were created by sequelize, I'm wondering how I can put them inside the src folder which currently contains the folders with the same names? like config. or can I just put them inside another folder so that there won't be any confusion between files of src and sequelize